### ISSUE TITLE
Combobox does not work on VoiceOver Safari

### ISSUE THREAD

**Issue by eirikbacker** [digdir/designsystemet#2322]
...since Safari does not support `aria-activedecendant`. We should consider
https://u-elements.github.io/u-elements/elements/u-datalist
https://u-elements.github.io/u-elements/elements/u-tags

See also #1732

**Comment by mrosvik** [comment]
We’re closing this issue because `Combobox` is deprecated. It has been replaced by `Suggestion`.
